packages/data/src/redux-store/index.js

```diff
@@ -221,12 +221,14 @@ export default function createReduxStore( key, options ) {
 					get: ( target, prop ) => {
 						return (
 							mapSelectors(
-								mapValues(
-									privateSelectors,
-									( selector ) =>
-										( state, ...args ) =>
-											selector( state.root, ...args )
-								),
+								mapValues( privateSelectors, ( selector ) => {
+									if ( selector.isRegistrySelector ) {
+										selector.registry = registry;
+									}
+
```

Comment:
This bit of code was missing. It's the same as what's on line 208 above. Without it you get an error when using `createRegistrySelector` for a private selectors.